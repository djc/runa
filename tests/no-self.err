tests/no-self.rns [2.5]: missing 'self' argument
    def __init__():
    ^
