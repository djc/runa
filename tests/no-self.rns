class test:
	def __init__():
		return
