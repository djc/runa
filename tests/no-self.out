tests/no-self.rns [2.2]: missing 'self' argument
    def __init__():
    ^
